static/app/components/events/interfaces/spans/inlineDocs.tsx

```diff
@@ -123,9 +39,183 @@ class InlineDocs extends Component<Props, State> {
             }
           )}
         </p>
-      </div>
-    );
-  }
+      )}
+    </div>
+  );
+}
+
+function PythonDocs() {
+  return (
+    <DocumentationWrapper>
+      <p>
+        {tct(
+          "The Sentry SDK for Python does a very good job of auto instrumenting your application. Here is a short introduction on how to do custom performance instrumentation. If you'd like to learn more, read our [link:custom instrumentation] documentation.",
+          {
+            link: (
+              <ExternalLink href="https://docs.sentry.io/platforms/python/tracing/instrumentation/custom-instrumentation/" />
+            ),
+          }
+        )}
+      </p>
+      <p>
+        <strong>{t('Adding a Transaction')}</strong>
+      </p>
+      <p>
+        {t(
+          'Adding transactions will allow you to instrument and capture certain regions of your code.'
+        )}
+      </p>
+      <p>
+        {t(
+          "If you're using one of Sentry's SDK integrations, transactions will be created for you automatically."
+        )}
+      </p>
+      <p>
+        {tct(
+          'The following example creates a transaction for an expensive operation (in this case, [code:eat_pizza]), and then sends the result to Sentry:',
+          {code: <code />}
+        )}
+      </p>
+      <StyledCodeSnippet language="python" dark>
+        {`import sentry_sdk
+
+def eat_slice(slice):
+    ...
+
+def eat_pizza(pizza):
+    with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
+        while pizza.slices > 0:
+            eat_slice(pizza.slices.pop())
+`}
+      </StyledCodeSnippet>
+      <p>
+        <strong>{t('Adding More Spans to the Transaction')}</strong>
+      </p>
+      <p>
+        {t(
+          'If you want to have more fine-grained performance monitoring, you can add child spans to your transaction, which can be done by either:'
+        )}
+      </p>
+      <ul>
+        <li>{t('Using a context manager or')}</li>
+        <li>{t('Using a decorator, (this works on sync and async functions)')}</li>
+      </ul>
```

Comment:
I think this is fine since you are mimicking how the docs were, but I was wondering when we should or shouldn't use our `<List />` component